[2022-06-05 20:35:38,376] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T23:34:56.495603+00:00 [queued]>
[2022-06-05 20:35:38,393] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T23:34:56.495603+00:00 [queued]>
[2022-06-05 20:35:38,393] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 20:35:38,393] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 20:35:38,393] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 20:35:38,415] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): populate_table> on 2022-06-05 23:34:56.495603+00:00
[2022-06-05 20:35:38,423] {standard_task_runner.py:52} INFO - Started process 171694 to run task
[2022-06-05 20:35:38,427] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'populate_table', 'scheduled__2022-06-05T23:34:56.495603+00:00', '--job-id', '1026', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp8mylju83', '--error-file', '/tmp/tmpl_i1h2hc']
[2022-06-05 20:35:38,428] {standard_task_runner.py:80} INFO - Job 1026: Subtask populate_table
[2022-06-05 20:35:38,506] {task_command.py:370} INFO - Running <TaskInstance: census.populate_table scheduled__2022-06-05T23:34:56.495603+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 20:35:38,606] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=populate_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T23:34:56.495603+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T23:34:56.495603+00:00
[2022-06-05 20:35:38,822] {python.py:173} INFO - Done. Returned value was: None
[2022-06-05 20:35:38,846] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=populate_table, execution_date=20220605T233456, start_date=20220605T233538, end_date=20220605T233538
[2022-06-05 20:35:38,880] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 20:35:38,936] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
